Clicker/ViewControllers/DraftsViewController.swift

```diff
@@ -57,7 +57,8 @@ class DraftsViewController: UIViewController {
         view.addSubview(visualEffectView)
         
         backButton = UIButton()
-        backButton.setImage(#imageLiteral(resourceName: "whiteExit"), for: .normal)
+        backButton.setImage(UIImage(named: "darkexit")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
can you add `let backButtonImageName = "darkexit"` to the top of file and use that instead